das/dastree/dastree.go

```diff
@@ -120,20 +153,22 @@ func ValidHash(hash bytes32, preimage []byte) bool {
 	return false
 }
 
-func Content(root bytes32, oracle func(bytes32) []byte) ([]byte, error) {
-	// Reverses hashes to reveal the full preimage under the root using the preimage oracle.
-	// This function also checks that the size-data is consistent and that the hash is canonical.
-	//
-	// Notes
-	//     1. Because we accept degenerate dastrees, we can't check that single-leaf trees are canonical.
-	//     2. For any canonical dastree, there exists a degenerate single-leaf equivalent that we accept.
-	//     3. We also accept old-style flat hashes
-	//     4. Only the committee can produce trees unwrapped by this function
-	//     5. Only the replay binary calls this
-	//
+// Reverses hashes to reveal the full preimage under the root using the preimage oracle.
+// This function also checks that the size-data is consistent and that the hash is canonical.
+//
+// Notes
+//     1. Because we accept degenerate dastrees, we can't check that single-leaf trees are canonical.
+//     2. For any canonical dastree, there exists a degenerate single-leaf equivalent that we accept.
+//     3. We also accept old-style flat hashes
+//     4. Only the committee can produce trees unwrapped by this function
+//     5. When the replay binary calls this, the oracle function supplied must not be able to return an error.
```

Comment:
~must not be able to return an error~ `must be infallible`

Reply:
Better wording, thanks.